doc/classes/Node.xml

```diff
@@ -234,6 +234,13 @@
 				If the node is not inside the tree, returns [code]false[/code] no matter the value of [member process_mode].
 			</description>
 		</method>
+		<method name="clear_children">
+			<return type="void" />
+			<param index="0" name="include_internal" type="bool" default="false" />
+			<description>
+				Calls [method queue_free] on all children.
```

Comment:
This should contain an explanation of the `include_internal` parameter in case it is shown out of context. You can just copy it from `get_children`